dev-util/phpstorm/phpstorm-9.0.2.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit eutils versionator vcs-snapshot
+
+SLOT="0"
```

Comment:
I'm sorry if I missed this before but why did you move `SLOT` here? It's recommended to use standard variable order to make ebuild reading easier for others, that is:

    DESCRIPTION
    HOMEPAGE
    SRC_URI

    LICENSE
    SLOT
    KEYWORDS
    IUSE